content/kotlin/concepts/data-types/data-types.md

```diff
@@ -0,0 +1,390 @@
+---
+Title: 'Data-Types'
+Description: 'In Kotlin, a data type is a classification of types of values that can be stored and manipulated in a program.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Primitive Data-Type'
+  - 'Non Primitive Data-Type'
+  
```

Comment:
Remove the extra line break in the metadata.

Reply:
@dakshdeepHERE: You marked this resolved without addressing the issue.

Reply:
Done I thought I was suppose to remove the extra line below this meta data. Sorry.